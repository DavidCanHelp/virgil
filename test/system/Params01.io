% a bakedFDA c -def 
stdout:"0:a 1:backedFDA 2:c 3:-def"
exit:0
