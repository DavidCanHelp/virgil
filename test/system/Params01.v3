// tests that params work
component Params01 {
	def main(a: Array<string>) {
		for (i = 0; i < a.length; i++) {
			if (i > 0 ) System.puts(" ");
			System.puti(i);
			System.puts(":");
			System.puts(a(i));
		}
	}
}
