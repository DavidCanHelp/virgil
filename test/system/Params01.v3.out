0:a 1:bakedFDA 2:c 3:-def